kymatio/scattering3d/backend/tensorflow_backend.py

```diff
@@ -178,8 +108,12 @@ def concatenate(arrays, L):
 
 backend.name = 'tensorflow'
 backend.cdgmm3d = cdgmm3d
-backend.fft = fft
-backend.modulus = complex_modulus
+backend.fft = FFT(lambda x: tf.signal.fft3d(x, name='fft3d'),
+                  lambda x: tf.signal.fft3d(tf.cast(x, tf.complex64), name='rfft3d'),
+                  lambda x: tf.signal.ifft3d(x, name='ifft3d'),
+                  lambda x: tf.math.real(tf.signal.ifft3d(x, name='irfft3d')),
+                  lambda x: None, real_check, complex_check)
+backend.modulus = Modulus()
```

Comment:
Is this ok? Again, what's the different here between the general and the 3D versions?

Reply:
Explain?

Reply:
I'm not sure. Tensorflow 3d modulus is the same as the modulus function in the general backend. 

I have a feeling for the sk cuda backend the modulus function has a different cuda routine for each dimension, but I am not sure on the details. Let me do some digging.